docs/concepts/model-serving.md

```diff
@@ -0,0 +1,32 @@
+(model_serving)=
+# Model Serving
+
+MLRun model serving allow composition of multi-stage real-time pipelines that include data manipulation and execution of models. The architecture allows high scalability and low latency performance.
+
+## Basic Model Serving
+
+The most basic model serving capability is deployment of a single model. To do that one has to:
+1. Create an MLRun function of type `serving` that implements a serving class with the `load` and `predict` methods. MLRun function marketplace comes with a range of such functions that support most common frameworks
+2. Add the model to the function, using the `add_model` method.
+3. Deploy the model, using the `deploy` model.
+
+This results with a single model endpoint that can execute the model and return the model prediction.
+
+Optionally, one can create a mock server, which runs the model as an in-memory object within the user's development environment. This allows testing the model without deploying it.
```

Comment:
Optionally, you can create a [mock server](../serving/getting-started.html#test-the-function), which runs the model as an in-memory object withinyour development environment. This allows testing the model without deploying it.